### ISSUE TITLE
Add to collection doesnt recognize parent model id

### ISSUE THREAD

**Issue by sfielder** [balderdashy/sails#2265]
Attempting to create a related stream record from the parent record item based on the this doc (http://sailsjs.org/#/documentation/reference/blueprint-api/Add.html) 

Is this feature fully baked and/or is there configuration required to enable the blueprints for related models?

POST item/542616d7dc08fb0200dbc84a/streams/

body:

{
  "name" : "temperature",
  "value" : 76,
  "unit" : "farenheit °" 
 }

Responds with validation error and does not pickup the item id posted in the request.

{
  "error": "E_VALIDATION",
  "status": 400,
  "summary": "1 attribute is invalid",
  "model": "Stream",
  "invalidAttributes": {
    "item": [
      {
        "rule": "required",
        "message": "\"required\" validation rule failed for input: null"
      }
    ]
  }
}

Model association reference..
streams: {
      collection: 'stream',
      via: 'item'
    }

item: {
			model: 'item',
			required: true
		},

**Comment by CWyrtzen** [comment]
Are ya'll still having this problem with subsequent releases?

**Comment by sailsbot** [comment]
Thanks for posting, @sfielder.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
